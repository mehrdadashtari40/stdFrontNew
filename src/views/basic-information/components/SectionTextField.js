import React, {Component} from "react";

export default class SectionTextField extends Component {
	render() {
		return (
		      <section className={this.props.gridClass}>
                      <label className="input">
                          <i className={'icon-prepend fa '+this.props.fontAwsName}/>
                          <input type={this.props.type} value={this.props.value} id={this.props.id}
                                 name={this.props.name} placeholder={this.props.placeholder} />
                      </label>
                  </section>
            )
	}
}
