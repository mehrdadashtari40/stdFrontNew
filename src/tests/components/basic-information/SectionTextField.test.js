import React from 'react';
import { shallow } from 'enzyme';

import SectionTextField from '../../../views/basic-information/components/SectionTextField';

jest.mock('react-i18next', () => ({
	withTranslation: () => Component => {
		Component.defaultProps = { ...Component.defaultProps, t: () => '' };
		return Component;
	},
}));

test('Make Snapshot file , SectionTextField Component', () => {
	const wrapper = shallow(<SectionTextField />);

	expect(wrapper).toMatchSnapshot();
});
