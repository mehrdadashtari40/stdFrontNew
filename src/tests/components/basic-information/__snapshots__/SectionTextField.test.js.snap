// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make Snapshot file , SectionTextField Component 1`] = `
<section>
  <label
    className="input"
  >
    <i
      className="icon-prepend fa undefined"
    />
    <input />
  </label>
</section>
`;
